src/test/java/nextstep/core/member/acceptance/MemberAcceptanceTest.java

```diff
@@ -81,35 +82,25 @@ class 성공 {
             @Test
             void 정상적인_토큰으로_조회() {
                 // given
-                var 이메일 = "admin@email.com";
-                var 비밀번호 = "password";
-                var 나이 = 20;
-                회원_생성_요청(이메일, 비밀번호, 나이);
+                회원_생성_요청(SMITH);
 
-                var 발급된_토큰 = 성공하는_토큰_발급_요청(이메일, 비밀번호);
+                var 발급된_토큰 = 성공하는_토큰_발급_요청(SMITH);
 
                 // when
                 var 토큰으로_전달받은_회원_정보 = 성공하는_토큰으로_회원정보_요청(발급된_토큰);
 
                 // then
-                회원_정보_확인(토큰으로_전달받은_회원_정보, 이메일, 나이);
+                회원_정보_확인(토큰으로_전달받은_회원_정보, SMITH);
```

Comment:
기존에는 이메일과 나이를 검증한다는 부분을 유추할 수 있었는데, 현재는 그런 부분이 생략된 것 같아요.
아래와 같이 드러내보면 어떨까요?.
```suggestion
                회원_정보_확인(토큰으로_전달받은_회원_정보, SMITH.email, SMITH.age);
```

Reply:
앗..
확실히 준오님께서 제안해주신 코드가 이해하기 쉬울 것 같습니다.

또, 작성된 영어 이름(`SMITH`)을 한글 이름(`김영호` 등..)으로 바꾸면 이해당사자(eg. 기획자, 프론트 개발자)가 봤을 때 이해하기 쉬울 것 같다는 생각이 듭니다. 😆 

다음 미션에 반영하겠습니다 👍